app/models/allinson_flex/profile_class.rb

```diff
@@ -4,7 +4,7 @@ module AllinsonFlex
   class ProfileClass < ApplicationRecord
     # @todo any before_destroy validation?
     # many to many relationship between class and context
-    has_and_belongs_to_many :profile_contexts
+    has_and_belongs_to_many :profile_contexts # this seems incorrect??
```

Comment:
Added this note, as has_and_belongs_to_many is not defined on both models, and additionally there is a defined table, ProfileClassContext as the intersection of these two classes.

Reply:
Could you update the 'this seems incorrect??' comment with the details provided in your PR comment? Having the extra context will be useful to us when we evaluate this later.